### PR TITLE
[IMP]10703 add module mrp_production_workcenter_line_reporting_excel

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @siyuancn @duanyp1991

File: mrp_production_workcenter_line_reporting_excel/README.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============================
+Excel report for Mrp Production
```
Comment (elicoidal) [inline mrp_production_workcenter_line_reporting_excel/README.rst]: Mrp => MRP

File: mrp_production_workcenter_line_reporting_excel/README.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+===============================
+
+This module extends the functionality of mrp and allows you to export excel report for Mrp Production
+with speific category during one specific period.
+
```
Comment (elicoidal) [inline mrp_production_workcenter_line_reporting_excel/README.rst:11]: add a screenshot

File: mrp_production_workcenter_line_reporting_excel/report/report_quantity.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+        records = data['records']
+        # table writer
+        ws = wb.add_sheet('Quantity')
+        # style
+        style = self._get_xls_format()
+        # start col index
+        col = 0
+        total_col = len(data['title'])
+
+        main_title = u"协邦铸业产品工序产量报告ریخته گری دولت گزارش تولید گام محصول"
```
Comment (elicoidal) [inline mrp_production_workcenter_line_reporting_excel/report/report_quantity.py]: Switch to English

File: mrp_production_workcenter_line_reporting_excel/report/report_scraped.py [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+        # table writer
+        ws = wb.add_sheet('Scraped')
+        # style
+        style = self._get_xls_format()
+        # start col index
+        col = 0
+        total_col = len(data['title'])
+
+        main_title = u"\
+        协邦铸业生产工序报废报告انجمن صنعت ریخته گری فرآیندهای تولید دولت قراضه گزارش"
```
Comment (elicoidal) [inline mrp_production_workcenter_line_reporting_excel/report/report_scraped.py]: English

File: mrp_production_workcenter_line_reporting_excel/wizard/report_quantity.py [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+    #      13 |          75 |          188 |    1110.000 | B1                |          300
+    #      13 |          75 |          188 |    1110.000 | A1                |          100
+    #         |             |           33 |       1.000 | Repairs workshop  |
+    #         |             |            4 |       3.000 | Assemble          |         1088
+    #         |             |           10 |       2.000 | Assemble          |
+    #         |             |           33 |       1.000 | Assembly workshop |
+    def _get_res_from_sql(self):
+        user_id = self._context['uid']
+
+        select_statement = """
```
Comment (elicoidal) [inline mrp_production_workcenter_line_reporting_excel/wizard/report_quantity.py]: This statement doesnot seem multicompany

File: mrp_production_workcenter_line_reporting_excel/wizard/report_quantity.py [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
+        for key, val in vals.items():
+            if len(val['process']) > length:
+                length = len(val['process'])
+
+        return length
+
+    def _get_table_titile(self, process_len):
+        # save the format in col:title
+        title = {
+            u'0': u"销售订单سفارش فروش",
```
Comment (elicoidal) [inline mrp_production_workcenter_line_reporting_excel/wizard/report_quantity.py]: English

File: mrp_production_workcenter_line_reporting_excel/wizard/report_scraped.py [hunk 7]
```diff
@@ -0,0 +1,295 @@
...
+        for key, val in vals.items():
+            if len(val['process']) > length:
+                length = len(val['process'])
+
+        return length
+
+    def _get_table_titile(self, process_len):
+        # save the format in col:title
+        title = {
+            u'0': u"销售订单سفارش فروش",
```
Comment (elicoidal) [inline mrp_production_workcenter_line_reporting_excel/wizard/report_scraped.py]: English

File: mrp_production_workcenter_line_reporting_excel/wizard/report_quantity.py (hunk 5 above)
Comment (siyuancn) [inline mrp_production_workcenter_line_reporting_excel/wizard/report_quantity.py]: There is a where statement below: where_statement = 'where mo.company_id = u.company_id' which check the company id.

Comment (elicoidal) [inline mrp_production_workcenter_line_reporting_excel/wizard/report_quantity.py]: @siyuancn you are right! I overlooked it

File: mrp_production_workcenter_line_reporting_excel/README.rst [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (report_xls, mrp_production_workcenter_line_reporting)
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/Elico-Corp/odoo/issues>`_.
```
Comment (elicoidal) [inline mrp_production_workcenter_line_reporting_excel/README.rst]: URL+newest template

**Review by elicoidal — CHANGES_REQUESTED** [review]
mostly README+Change Chinese/Farsi to English

File: mrp_production_workcenter_line_reporting_excel/README.rst (hunk 2 above)
Comment (noahzaozao) [reply mrp_production_workcenter_line_reporting_excel/README.rst:11]: where is the screenshot?

**Comment by noahzaozao** [comment]
@elicoidal please review this pr again

File: mrp_production_workcenter_line_reporting_excel/README.rst (hunk 2 above)
Comment (elicoidal) [reply mrp_production_workcenter_line_reporting_excel/README.rst:11]: should be taken ;)
Forget about it

**Comment by elicoidal** [comment]
travis needs some love? @noahzaozao

**Comment by noahzaozao** [comment]
@elicoidal this module was depend on another module https://github.com/Elico-Corp/odoo-addons/pull/41

**Pull request merged**